tests/0011-produce_batch.c

```diff
@@ -90,13 +90,22 @@ static void test_single_partition(void) {
         int failcnt = 0;
         int i;
         rd_kafka_message_t *rkmessages;
+        const int topicSuffixLength = 56, clientIdLength = 239;
+        char *topicSuffix, *clientId;
 
         SUB_TEST_QUICK();
 
         msgid_next = 0;
 
         test_conf_init(&conf, &topic_conf, 20);
 
+        clientId = (char *)malloc(clientIdLength + 1 * sizeof(char));
+        for (i = 0; i < clientIdLength; i++) {
+                clientId[i] = 'c';
+        }
+        clientId[clientIdLength] = '\0';
+        rd_kafka_conf_set(conf, "client.id", clientId, NULL, 0);
```

Comment:
Please use snake case for variables and add a parenthesis around `client_id_length + 1`. Cast isn't needed as `void *` can be assigned without cast.

Also add a comment that links to this issue.
```suggestion
        client_id = malloc((client_id_length + 1) * sizeof(char));
        for (i = 0; i < client_id_length; i++) {
                client_id[i] = 'c';
        }
        client_id[client_id_length] = '\0';
```

Reply:
Updated